dts/bindings/pinctrl/nxp,mcux-rt-pinctrl.yaml

```diff
@@ -91,42 +126,54 @@ child-binding:
           101 DSE_5_R0_5 — 32 Ohm @3.3V, 52 Ohm @1.8V
           110 DSE_6_R0_6 — 32 Ohm @3.3V, 43 Ohm @1.8V
           111 DSE_7_R0_7 — 26 Ohm @3.3V, 37 Ohm @1.8V
-      bias-pull-up:
+      bias-pull-up-value:
         required: false
-        type: int
+        type: string
+        default: "pull-up-47k-ohm"
         enum:
-          - 0
-          - 1
-          - 2
-          - 3
+          - "unused"
+          - "pull-up-47k-ohm"
+          - "pull-up-100k-ohm"
+          - "pull-up-22k-ohm"
         description: |
           Select the value of the pull up resistor present on this pin
           Corresponds to the PUS field in the IOMUXC peripheral
           00 Unused- no change will be applied to pin
           01 PUS_1_47K_Ohm_Pull_Up — 47K Ohm Pull Up
           10 PUS_2_100K_Ohm_Pull_Up — 100K Ohm Pull Up
-          11 PUS_3_22K_Ohm_Pull_Up — 22K Ohm Pull Up
-      slew-rate:
+          11 PUS_2_22K_Ohm_Pull_Up — 22K Ohm Pull Up
+
+      bias-pull-down-value:
         required: false
-        type: int
+        type: string
+        default: "pull-down-100k-ohm"
+        enum:
+          - "pull-down-100k-ohm"
+        description: |
+          Select the value of the pull up resistor present on this pin
+          Corresponds to the PUS field in the IOMUXC peripheral
+          00 PUS_0_100K_Ohm_Pull_Down - 100K Ohm Pull Down
```

Comment:
since there's only 1 pull-down value, is this required?

Reply:
No, it is not. I added it for the sake of consistency between pull up and pull down resistors, and because in my mind having that value visible in the generated DTS is useful. With the value included, a user can look at their output `zephyr.dts` and see the exact pull up or pull down resistor they have selected. It's not required however, and if you would prefer it was dropped I'll get rid of it.